be2-scala/src/main/scala/ch/epfl/pop/model/network/Validatable.scala

```diff
@@ -0,0 +1,7 @@
+package ch.epfl.pop.model.network
+
+import ch.epfl.pop.pubsub.graph.PipelineError
+
+trait Validatable {
```

Comment:
What a remarkably ugly word. I can't blame you, wiktionary says it exists and it seems appropriate :-)
https://en.wiktionary.org/wiki/validatable

Reply:
Hahah! ^^ If you find a better word, I'll happily change it! But I googled and couldn't find better :/

Reply:
Hopefully the "Answerer" graph component will not hurt your eyes too much when I push the next commit ^^'